source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,133 @@
+
+There is no configuration required for editing files within your /config directory. In order to enable access to further directories, editing the add-on configuration is required. See the add-on documentation for details. 
+
+---
+
+### Installing and using the File Editor add-on
+
+A more basic and light weight alternative to Visual Studio Code, the File Editor add-on provides access through Hass-Configurator, which is a web-based filesystem-browser and text-editor. YAML files are automatically checked for syntax errors while editing.
+
+There is no configuration required for editing files within your /config directory. In order to enable access to further directories, editing the add-on configuration is required. See the add-on documentation for details. 
+
+---
+
+## Home Assistant OS Snapshots
+
+Snapshots are a compressed archive file (.tar) of your Home Assistant configuration, in addition to any add-on data and configuration. These are easily made in Home Assistant OS from the Supervisor Snapshot panel and are also available as services which can be used in automations. They can be created from the UI or the CLI (see below) and are stored in the /backup directory.
+
+A full snapshot includes the following directories:
+
+* config
+* share
+* addons (these are manually installed or created add-ons, and not those installed from the store)
+* ssl
+* media
+
+A partial snapshot consists of any number of the above default directories and installed add-ons.
+
+### Making a Snapshot from the UI
+
+1. Go to Supervisor > Snapshots in the UI
+2. Provide a name for the snapshot.
+3. Choose full or partial.
+4. Choose to password protect or not. Password protected snapshots cannot easily be browsed outside of Home Assistant OS
+5. Click "Create" to begin the snapshot.
+
+### Restoring a Snapshot on a new install
+
+You can restore a snapshot during on a new installation during the onboarding process. Follow the link at the bottom of the account creation page to upload your snapshot from a previous installation.
+
+For restoring a snapshot at any other time, vist the Supervisor Snapshot panel in your UI and use the following steps:
+
+1. Select "Upload Snapshot" from the icon in the upper right of the page.
+1. Click on the folder icon to navigate to your snapshot .tar file
+
+When the upload is completed, you will be presented with the snapshot restore dialog for restoring it, and can then choose to restore in full or in part by manually selecting individual items.
+
+If a snapshot is significantly large, it can be faster and more efficient to make use of the Samba add-on in order to transfer files to the `/backup` directory. 
```

Comment:
Give a hint of what large is, ie >1GB.

Explain what the user should do after it transfers a snapshot to the `/backup` directory,